relay/prover.go

```diff
@@ -107,6 +111,16 @@ func (pr *Prover) Init(homePath string, timeout time.Duration, codec codec.Proto
 func (pr *Prover) SetRelayInfo(path *core.PathEnd, counterparty *core.ProvableChain, counterpartyPath *core.PathEnd) error {
 	pr.path = path
 	pr.counterpartyPath = counterpartyPath
+
+	if gauge, err := NewInt64Gauge(
+		fmt.Sprintf("lcp_updater_client_height.%s.%s", pr.originChain.ChainID(), counterparty.ChainID()),
```

Comment:
To make the metrics easier to visualize, it would be better to set chain IDs as attributes rather than embedding them in the metric name.